webapp/packages/core-connections/src/Administration/Connections/ConnectionForm/ConnectionFormController.ts

```diff
@@ -114,22 +120,30 @@ implements IInitializableController {
     return false;
   }
 
-  private valdiate() {
-    const status = { passed: true, errorMsg: '' };
-    const connectionName = this.model.connection.name?.trim();
-    if (!connectionName.length) {
-      status.errorMsg = "Field 'Name' can't be empty";
-    } else if (this.model.editing && this.isConnectionNameAlreadyExists(connectionName)) {
-      status.errorMsg = 'Connection with this name already exists';
+  private valdiate(config: ConnectionConfig) {
+    const validateByLength: Array<keyof ConnectionConfig> = ['name'];
+    const validationStatus: IValidationStatus = { status: true, errorMessage: '' };
+
+    for (const key of validateByLength) {
+      if (!config[key]?.length) {
+        validationStatus.errorMessage = `Field '${key}' can't be empty`;
+        break;
+      }
     }
 
-    status.passed = !status.errorMsg;
-    return status;
+    if (!validationStatus.errorMessage) {
+      if (this.model.editing && this.isConnectionNameAlreadyExists(config.name!)) {
+        validationStatus.errorMessage = 'Connection with this name already exists';
+      }
+    }
+
+    validationStatus.status = !validationStatus.errorMessage;
+    return validationStatus;
   }
```

Comment:
little mistake `valdiate` should be `validate`
code is too complicated with unnecessary functionality (array of fields for check for empty)
replace to `if else` chain to avoid unnecessary status check between validation tests